lib/src/main/java/com/auth0/android/lock/PasswordlessLockActivity.java

```diff
@@ -442,7 +442,7 @@ public void onOAuthAuthenticationRequest(OAuthLoginEvent event) {
         }
 
         Log.d(TAG, "Couldn't find an specific provider, using the default: " + WebAuthProvider.class.getSimpleName());
-        webProvider.start(this, event.getConnection(), authProviderCallback, WEB_AUTH_REQUEST_CODE);
+        webProvider.start(this, event.getConnection(), authParameters, authProviderCallback, WEB_AUTH_REQUEST_CODE);
```

Comment:
since `authParameters` is not modified for usage here, just use `options.getAuthenticationParameters()` (or null) and avoid to always make the deep copy above